src/components/editor/List.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import ReactList from 'react-list';
+import { Link } from 'react-router';
+
+export default class List extends React.Component {
```

Comment:
I'll figure something out about the whole list thing, and how I even code it and not using react-list etc.